examples/jsm/renderers/CSS2DRenderer.js

```diff
@@ -150,7 +150,15 @@ class CSS2DRenderer {
 
 					object.onBeforeRender( _this, scene, camera );
 
-					element.style.transform = 'translate(' + ( - 100 * object.center.x ) + '%,' + ( - 100 * object.center.y ) + '%)' + 'translate(' + ( _vector.x * _widthHalf + _widthHalf ) + 'px,' + ( - _vector.y * _heightHalf + _heightHalf ) + 'px)';
+					let transformString = 'translate(' + ( - 100 * object.center.x ) + '%,' + ( - 100 * object.center.y ) + '%)' + 'translate(' + ( _vector.x * _widthHalf + _widthHalf ) + 'px,' + ( - _vector.y * _heightHalf + _heightHalf ) + 'px)';
+					
+					if ( parameters.mode === 'scale' ) {
+
+						transformString += ' scale(' + camera.zoom + ', ' + camera.zoom + ')';
+						element.style.transform = transformString;
```

Comment:
The PR in its current state breaks all CSS2D examples since `element.style.transform` is only assigned in the `scale` mode now.